consolidate_metrics_.py

```diff
@@ -0,0 +1,48 @@
+
+import os
+import argparse
+import pandas as pd
+
+def consolidate_metrics(input_folder, output_csv_path):
+    # Initialize an empty dictionary to hold DataFrames for each AI model
+    model_data_dict = {}
+    csv_files = [f for f in os.listdir(input_folder) if f.endswith('.csv')]
```

Comment:
I guess specifying just the input folder might be better than using the models csv file used for test_models.py, as we might run testing multiple times on different models. But maybe we can make this recursive (my folder structure has the results csv files in separate nested directories).

Reply:
@pdc878 we just talked with Chris. Rather than searching for all .csv files, it would be safer to expect an input csv file that holds the list of result csv files to aggregate. You could actually use the same csv file that was used for test_models.py as input. That way no random csv files in the same folder will mess up how this script works.